extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/parent/MergeParentMetadata.java

```diff
@@ -36,9 +49,33 @@ public Date getTimestamp() {
         return timestamp;
     }
 
+    /**
+     * @return - The copy tool split time metadata for the table.
+     */
+    public Date getCopyToolInputTime() {
```

Comment:
so after talking with puja I see why The clone time is needed, but not the others